src/ooni/dns_injection_impl.hpp

```diff
@@ -37,9 +37,9 @@ class DNSInjectionImpl : public DNSTestImpl {
               std::function<void(report::Entry)> &&cb) {
         entry["injected"] = NULL;
         query("A", "IN", input, options["nameserver"],
-              [this, cb](dns::Response response) {
```

Comment:
This has now been addressed by a hotfix I pushed to repair building another branch.